pkg/minikube/cluster/cluster.go

```diff
@@ -272,3 +278,25 @@ func GetHostDockerEnv(api libmachine.API) (map[string]string, error) {
 	}
 	return envMap, nil
 }
+
+// GetHostLogs gets the localkube logs of the host VM.
+func GetHostLogs(api libmachine.API) (string, error) {
+	dne := "Does Not Exist"
+	exists, err := api.Exists(constants.MachineName)
```

Comment:
This code (to check if the machine exists, and if it does, try to load the machine from the api, etc...) seems generic enough for every minikube operation, so I think that you should break that out to a separate function that's used everywhere